vshard/util.lua

```diff
@@ -236,6 +236,19 @@ local function table_minus_yield(dst, src, interval)
     return dst
 end
 
+--
+-- Append all items of src to the end of dst. It is assumed both tables are
+-- arrays.
+--
+local function table_concat(dst, src)
+    local dst_i = #dst + 1
+    for _, v in ipairs(src) do
+        dst[dst_i] = v
+        dst_i = dst_i + 1
+    end
```

Comment:
```
tarantool> dst = {1, 2, 3}
---
...

tarantool> src = {4, 5, 6}
---
...

tarantool> table.move(src, 1, #src, #dst + 1, dst)
---
- - 1
  - 2
  - 3
  - 4
  - 5
  - 6
...

tarantool> dst
---
- - 1
  - 2
  - 3
  - 4
  - 5
  - 6
...
```

Reply:
Thanks, I took this expression as the body of `table_extend`.